src/redux/features/books/booksSlice.js

```diff
@@ -12,7 +31,8 @@ const booksSlice = createSlice({
       state.books.push(action.payload);
     },
     removeBook: (state, action) => {
-      state.books.filter((book) => book.id !== action.payload);
+      const filteredBooks = state.books.filter((book) => book.item_id !== action.payload);
+      return { ...state, books: filteredBooks };
     },
   },
 });
```

Comment:
- [OPTIONAL] I know it isn't in your project requirement, but it would be nice to use the `categoriesSlice` in the React component. That way, you can dispatch the `checkStatus` action in the Categories component and immediately display the **_'Under Construction'_** text. 👍 💻

Reply:
Thanks for your wise advice @Gambit142, I will apply it. 👍